src/app/(pages)/clients/[client]/components/Form/tabs/GrantTypes/index.tsx

```diff
@@ -9,7 +9,9 @@ export default async function GrantTypes(props: Readonly<GrantTypesProps>) {
   const openIdConfiguration = await fetchOpenIdConfiguration();
   const active_grant_types = props.grant_types ?? [];
   const { grant_types_supported } = openIdConfiguration;
-  const grant_types = grant_types_supported.map(gt => {
+  const grant_types = (
+    grant_types_supported ?? ["authorization_code", "implicit"]
```

Comment:
With new client creation page we want to support only one grant type at the time, as mutual exclusive option.
Even though the grant types field is a list, it must be exactly one element long.

Reply:
Sorry my fault, I thought it was related to the client creation but we are talking about the client editing/visualization.
I'm not sure if we want to add not one, but two fallback options in case the client has no grant type after its creation.
In any case, if I remember correctly, we don't want to support implicit credential
@giacomini @enricovianello

Reply:
I agree that implicit must be removed from the well known (that's tautological), but still I do not understand why we want to hardcode the `implicit` fallback value if don't want to support it at all. Ok to make TypeScript happy, but I think it will be happy also without the `implicit` value :D